pkg/policy/api/egress.go

```diff
@@ -199,6 +209,16 @@ type EgressDenyRule struct {
 	//
 	// +kubebuilder:validation:Optional
 	ToPorts PortDenyRules `json:"toPorts,omitempty"`
+
+	// ICMP is a list of ICMP-type identified by type number
+	// which the endpoint subject to the rule is not allowed to connect to.
```

Comment:
```suggestion
	// ICMP is a list of ICMP rules identified by type number
	// which the endpoint subject to the rule is not allowed to connect to.
```

Would that sound better?